creator-cards/egvijayanand.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "Vijay Anand E G",
+  "sectionTitle": null,
+  "countryCode": "in",
+  "slogan": "",
+  "type": "CREATOR",
+  "bio": ".NET Geek | Blogger | Content Creator | .NET MAUI, Blazor, C# | He/Him",
+  "tags": ".NET, C#, .NET MAUI, ASP.NET, Blazor",
+  "github": "https://github.com/egvijayanand",
+  "socials": {
+    "linkedin": "https://www.linkedin.com/in/egvijayanand",
+    "twitter": "https://twitter.com/egvijayanand",
+    "github": "https://github.com/egvijayanand",
+    "nuget": "https://www.nuget.org/profiles/egvijayanand"
+  },
+  "channels": [
+    {
+      "name": "Blog",
+      "url": "https://egvijayanand.in/"
+    },
+    {
+      "name": "Site",
+      "url": "https://egvijayanand.in/"
+    }
+  ],
+  "feeds": [
```

Comment:
If all these feeds return the same data please only give us one, atom preferred. 
Also we changed the type recently to be either dotnet or personal while the later is used if your feed
contains things that are not .NET related.

Reply:
Pushed a commit with those changes.